sysutils/upt/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                upt
+version             0.8
+revision            0
+
+categories-prepend  sysutils
+platforms           darwin
+supported_archs     noarch
+license             BSD
+maintainers         {@korusuke somaiya.edu:karan.sheth} openmaintainer
+
+description         Package software from any package manager to any distribution
+long_description    ${description}
+
+homepage            https://framagit.org/upt/upt
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            ${python.rootname}-${version}
+
+checksums           sha256  ecb4b8da0e2e6a5f6926dc41cc45fd3923e8dfb6a9a8aafaa9149b5144d8898c \
+                    rmd160  3d3aade4ec513943b4fc8f12042a3f4328f60102 \
+                    size    25792
+
+python.default_version  37
+
+depends_lib-append \
+                port:py${python.version}-spdx-lookup \
+                port:py${python.version}-setuptools
+
+depends_run-append \
+                port:py${python.version}-upt-cpan \
+                port:py${python.version}-upt-macports \
+                port:py${python.version}-upt-pypi \
+                port:py${python.version}-upt-rubygems
+
+test.run        yes
+test.cmd        ${python.bin} -m unittest
+test.target
+test.env        PYTHONPATH=${worksrcpath}/build/lib
+
+livecheck.type  none
```

Comment:
switching off the livecheck was done for the original subports, but this line should be removed here as we otherwise can't use ```port livecheck upt``` anymore for updates.